src/api.ts

```diff
@@ -92,23 +113,41 @@ router.post('/logout', (req, res) => {
 router.get('/allRoles', getAllRoles);
 
 // Members
-router.get('/allMembers', allMembers);
-router.post('/setMember', setMember);
-router.post('/deleteMember', deleteMember);
+router.get('/allMembers', async (req, res) => {
+  let handled = await allMembers(req, res); res.status(handled.status).json(handled);
```

Comment:
I think this line would be better off as two separate lines for each of the routes (lines 117, 120, 123, etc.)
i.e.<br /> ` let handled = ...`<br /> `res.status(...`